Framework/DataHandling/inc/MantidDataHandling/LoadILLTOF2.h

```diff
@@ -70,6 +70,7 @@ class DLLExport LoadILLTOF2 : public API::IFileLoader<Kernel::NexusDescriptor> {
   static double calculateError(double in) { return sqrt(in); }
 
   API::MatrixWorkspace_sptr m_localWorkspace;
+  bool m_convertToTOF;
```

Comment:
It looks like this is only used ion the exec method, so it does not need to be a class level variable